schema-loader/src/test/java/com/scalar/db/schemaloader/SchemaLoaderTest.java

```diff
@@ -916,6 +916,616 @@ public void tearDown() throws Exception {
     }
   }
 
+  // For backward compatibility with the old method signatures.
```

Comment:
Copied these test cases from the original test code right before https://github.com/scalar-labs/scalardb/pull/2747 was merged.